packages/xod-client/src/user/containers/AccountPane.jsx

```diff
@@ -0,0 +1,88 @@
+import R from 'ramda';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { foldMaybe } from 'xod-func-tools';
+
+import Button from '../../core/components/Button';
+
+import * as Actions from '../actions';
+import * as Selectors from '../selectors';
+
+import AuthForm from '../components/AuthForm';
+
+const AccountPane = ({
+  user,
+  isVisible,
+  compilationsLeft,
+  actions,
+  isAuthorising,
+  authError,
+}) => {
+  if (!isVisible) return null;
+
+  return (
+    <div className="AccountPane">
+      <a
+        role="button"
+        tabIndex="0"
+        className="close-button"
+        onClick={actions.toggleAccountPane}
+      >
+        &times;
+      </a>
+
+      {foldMaybe(
+        <div className="username">Guest XODer</div>,
+        ({ username }) => (
+          <div>
+            <div>Logged in as</div>
+            <div className="username">{username}</div>
+          </div>
+        ),
+        user
+      )}
+
+      <div className="compilationsLeft">
+          Compilations left: {compilationsLeft == null ? 'currently offline' : compilationsLeft}
```

Comment:
Recently I realized that “1000 compilations left” sounds like it is the limit ever. But it is for today. I think we should emphasize it. So I’d add a “Daily quotas” heading right above “Compilations left”